### PR TITLE
MAT-312 allowance method for erc20

### PR DESCRIPTION
**Pull request by jdkanani**

### CODE REVIEW

File: examples/node/allowance-ERC20.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const Matic = require('maticjs').default
+const config = require('./config')
+
+const token = config.ROPSTEN_TEST_TOKEN // test token address
```
Comment (0xAshish) [inline examples/node/allowance-ERC20.js]: `ROPSTEN_TEST_TOKEN` should be `PARENT_TEST_TOKEN` why keep specific network name in the codebase.

Comment (jdkanani) [reply examples/node/allowance-ERC20.js]: @arthcp can you take this up?

Comment (arthcp) [reply examples/node/allowance-ERC20.js]: Sure! I'll make the changes.

**Comment by jdkanani** [comment]
@saylip review & merge

**Pull request merged**